### PR TITLE
Fixing the app.css problem. No need to use BASE_URL

### PR DESCRIPTION
**Pull request by moxiegirl**
Using the same partials that work elsewhere.

Signed-off-by: Mary Anthony <mary@docker.com>

### CODE REVIEW

File: layouts/404.html [hunk 1]
```diff
@@ -1,147 +1,4 @@
...
-          <li><a href="#">Sign in</a></li>
-          <li><a href="#">Sign up</a></li>
-        </ul>
-      </aside>
-      <!-- close the off-canvas menu -->
-      <a class="exit-off-canvas"></a>
-
-
-
+{{ partial "header.html" . }}
```
Comment (SvenDowideit) [inline layouts/404.html]: this won't work, as the header.html uses links that hugo makes relative.

**Comment by SvenDowideit** [comment]
dup of #166?

**Comment by SvenDowideit** [comment]
dupe of #165?

File: layouts/404.html (hunk 1 above)
Comment (moxiegirl) [inline layouts/404.html]: This is the same header used in every page that we are generating. How come those pages are not breaking?

Comment (moxiegirl) [inline layouts/404.html]: I've tested and verified this works on docs-stage.docker.com. Try it for yourself

Comment (dnephin) [inline layouts/404.html]: @SvenDowideit is right, check this out: http://docs-stage.docker.com/engine/userguide/networking/bogus

I think we either need to set BaseURL to not have a scheme (`//docs.docker.com`), or we could just use a very simple 404 page without this header (which makes it a lot easier). I think many sites do this.

**Comment by moxiegirl** [comment]
@SvenDowideit this is into Master the other is into Hugo branch

**Pull request closed without merging**